api_tests/nodes/views/test_node_addons.py

```diff
@@ -685,6 +685,17 @@ def _settings_kwargs(self, node, user_settings):
         }
 
 
+class MockConnection():
```

Comment:
Minor: since we're on Python 2, "top-level" classes should still inherit `object`. Shouldn't make a difference in this case.